test/smoke-test.js

```diff
@@ -36,12 +37,64 @@ describe('smoke test', () => {
     const client = togglClient();
 
     const workspaces = await client.workspaces.list();
+    const detailsReport = await client.reports.details(workspaces[0].id, {
+      start_date: dayjs().subtract(1, 'week').format('YYYY-MM-DD'),
+    });
+
+    debug(detailsReport[0]);
+    expect(detailsReport).to.exist.to.be.an('array');
+    expect(detailsReport[0]).to.have.property('user_id');
+    expect(detailsReport[0]).to.have.property('username');
+    expect(detailsReport[0]).to.have.property('project_id');
+    expect(detailsReport[0]).to.have.property('task_id');
+    expect(detailsReport[0]).to.have.property('description');
+    expect(detailsReport[0]).to.have.property('time_entries');
+    expect(detailsReport[0]).to.have.property('row_number');
+  });
+
+  it.skip('should throw an error if a start date is not provided with a details report', async () => {
+    const client = togglClient();
 
-    const detailsReport = await client.reports.details(workspaces[0].id);
+    const workspaces = await client.workspaces.list();
+    // FIXME - this assertion is not working
+    expect(async function () {
+      await client.reports.details(workspaces[0].id);
+    }).to.throw('Error: The parameters must include start_date');
+  });
```

Comment:
@mcndt @saintedlama I'm not sure how this unit test is supposed to be written, but I'm trying to ensure that the exception handling is working